client/my-sites/purchases/controller.js

```diff
@@ -23,7 +24,12 @@ export function redirectToPurchases( context ) {
 }
 
 export const purchases = ( context, next ) => {
-	context.primary = <Purchases />;
+	context.primary = (
+		<>
+			<QueryConciergeInitial />
```

Comment:
I think it's better to pull the Query* component into the `Purchases` component itself, as it's much easier to see how its being used in context. Pushing it up into the controller feels like it makes the wiring even more magical, and may make it more complicated to work out how the `Purchases` component lifecycle will interact with the query component.

Reply:
I think I tried it there originally and it didn't work, hence putting it here. I will retry though.

Reply:
Well, it's working after I moved down into the `Purchases` component, so will commit a change.